src/Wallabag/CoreBundle/Resources/views/themes/baggy/Static/howto.html.twig

```diff
@@ -11,7 +11,7 @@
     </ul>
     <h3>{{ 'howto.top_menu.browser_addons'|trans }}</h3>
     <ul>
-        <li><a href="https://addons.mozilla.org/firefox/addon/wallabag/" target="_blank">{{ 'howto.browser_addons.firefox'|trans }}</a></li>
+        <li><a href="https://addons.mozilla.org/fr/firefox/addon/wallabag-v2/" target="_blank">{{ 'howto.browser_addons.firefox'|trans }}</a></li>
```

Comment:
Why using the french URL?
It should be better to avoid that language string :wink: 

https://addons.mozilla.org/firefox/addon/wallabag-v2/